setting_up_firely_server/configuration/sqlserver/db_sql.rst

```diff
@@ -7,6 +7,10 @@
 Using SQL server
 ================
 
+.. note::
+
+  The features described on this page are available in **all** :ref:`Firely Server editions <vonk_overview>`.
+
```

Comment:
SQLite and Memory are not explicitly mentioned in the editions page, but it would be consistent to add the same note to those repos as well.